packages/shared/test/sdk-versioning.test.ts

```diff
@@ -257,7 +254,7 @@ describe("payload scrubbing", () => {
           {
             condition: {
               id: {
-                $in: [],
+                $in: ["4", "5", "6"],
```

Comment:
This test case represents the main risk with this PR, namely that any large saved groups people have made since #2616 shipped will start to appear in payloads that they weren't before. @jdorn is that a behavior we're willing to accept or do we need to address that?